docs/man/ctags-lang-elm.7.rst

```diff
@@ -0,0 +1,152 @@
+	Result	input.elm	/^  , Result(..)$/;"	t	module:MyMod	roles:imported
+	MyList	input.elm	/^  , MyList(Empty)$/;"	t	module:MyMod	roles:imported
+	Empty	input.elm	/^  , MyList(Empty)$/;"	c	type:MyMod.MyList	roles:imported
+
+Namespaces
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Namespaces are tagged and their role is "def".
+
+.. code-block:: Elm
+
+	module AMod exposing (..)
+	
+	import MyImport as NSpace exposing (impFunc)
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+r"
+
+.. code-block:: tags
+
+	AMod	input.elm	/^module AMod exposing (..)$/;"	m	roles:def
+	NSpace	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	n	module:AMod	roles:def	moduleName:MyImport
+	MyImport	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	m	roles:imported
+	impFunc	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	f	module:MyImport	roles:imported
+
+Type descriptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Constructors will have type descriptions. So will any top level function
+that has a type annotation.
+
+.. code-block:: Elm
+
+	funcA : Int -> Int
+	funcA a = a + 1
+	
+	type B
+	    = B1Cons
+	      { x : Float
+	      , y : Float
+	      }
+	    | B2Cons String Integer
+	    | B3Cons
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+r"
+
+.. code-block:: tags
+
+	funcA	input.elm	/^funcA a = a + 1$/;"	f	typeref:description:Int -> Int	roles:def
+	B	input.elm	/^type B$/;"	t	roles:def
+	B3Cons	input.elm	/^    | B3Cons$/;"	c	type:B	typeref:description:	roles:def
+	B2Cons	input.elm	/^    | B2Cons String Integer$/;"	c	type:B	typeref:description:String -> Integer -> B	roles:def
+	B1Cons	input.elm	/^    = B1Cons$/;"	c	type:B	typeref:description:{ x : Float , y : Float } -> B	roles:def
+
+Function parameter lists
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Function parameter lists can be extracted into the tags file
+signature field. They are not really function signatures, but
+it's the closest concept available in ctags.
+Use "--fields=+S".
+
+.. code-block:: Elm
+
+    funcA a1 a2 =
+        a1 + a2
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+rS"
+
+.. code-block:: tags
+
+    funcA	input.elm	/^funcA a1 a2 =$/;"	f	signature:a1 a2	roles:def
+
+KNOWN LIMITATIONS
+-----------------
+The ctags signature field is used for function parameter lists, even
+though it's not an idea field. See above.
+
+Elm requires all statements at the same logical level to have the
+same indentation. If there is additional indentation that line is part
+of the previous one. Therefore without over-complicating the
+PEG parser we have the following limitations...
+
+Sometimes functions in let/in blocks will be omitted.
+
+Functions in let/in blocks will be marked as being in the scope of their
+outer function, regardless of how deeply nested the let/in block is.
+
+Functions in let/in blocks won't have type descriptions.
+
+SEE ALSO
+--------
+:ref:`ctags(1) <ctags(1)>`, :ref:`ctags-client-tools(7) <ctags-client-tools(7)>`
```

Comment:
Thank you for writing this man page, especially about the embedded test cases.
These test cases can be run with `make man-test`.
I know `man-test` is not sophisticated. https://github.com/universal-ctags/ctags/issues/3003
However, I cannot find time for improving.
So, you need the following changes to your test cases:

```diff
diff --git a/man/ctags-lang-elm.7.rst.in b/man/ctags-lang-elm.7.rst.in
index 7c3c58403..e69e3aeee 100644
--- a/man/ctags-lang-elm.7.rst.in
+++ b/man/ctags-lang-elm.7.rst.in
@@ -35,10 +35,12 @@ Imported items are tagged, but their role is "imported", not "def".
 Imported items are marked as being in the scope of their own module,
 not the module that's doing the importing.
 
+"input.elm"
+
 .. code-block:: Elm
 
 	module SomeMod exposing (..)
-	
+
 	import MyMod exposing
 	  ( map
 	  , Maybe
@@ -47,7 +49,7 @@ not the module that's doing the importing.
 	  )
 
 "output.tags"
-with "--sort=no --extras=+r --fields=+r"
+with "--options=NONE -o - --sort=no --extras=+r --fields=+r input.elm"
 
 .. code-block:: tags
 
@@ -64,14 +66,16 @@ Namespaces
 
 Namespaces are tagged and their role is "def".
 
+"input.elm"
+
 .. code-block:: Elm
 
 	module AMod exposing (..)
-	
+
 	import MyImport as NSpace exposing (impFunc)
 
 "output.tags"
-with "--sort=no --extras=+r --fields=+r"
+with "--options=NONE -o - --sort=no --extras=+r --fields=+r input.elm"
 
 .. code-block:: tags
 
@@ -86,11 +90,13 @@ Type descriptions
 Constructors will have type descriptions. So will any top level function
 that has a type annotation.
 
+"input.elm"
+
 .. code-block:: Elm
 
 	funcA : Int -> Int
 	funcA a = a + 1
-	
+
 	type B
 	    = B1Cons
 	      { x : Float
@@ -100,15 +106,15 @@ that has a type annotation.
 	    | B3Cons
 
 "output.tags"
-with "--sort=no --extras=+r --fields=+r"
+with "--options=NONE -o - --sort=no --extras=+r --fields=+r input.elm"
 
 .. code-block:: tags
 
 	funcA	input.elm	/^funcA a = a + 1$/;"	f	typeref:description:Int -> Int	roles:def
 	B	input.elm	/^type B$/;"	t	roles:def
-	B3Cons	input.elm	/^    | B3Cons$/;"	c	type:B	typeref:description:	roles:def
-	B2Cons	input.elm	/^    | B2Cons String Integer$/;"	c	type:B	typeref:description:String -> Integer -> B	roles:def
 	B1Cons	input.elm	/^    = B1Cons$/;"	c	type:B	typeref:description:{ x : Float , y : Float } -> B	roles:def
+	B2Cons	input.elm	/^    | B2Cons String Integer$/;"	c	type:B	typeref:description:String -> Integer -> B	roles:def
+	B3Cons	input.elm	/^    | B3Cons$/;"	c	type:B	typeref:description:B	roles:def
 
 Function parameter lists
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@@ -118,13 +124,15 @@ signature field. They are not really function signatures, but
 it's the closest concept available in ctags.
 Use "--fields=+S".
 
+"input.elm"
+
 .. code-block:: Elm
 
     funcA a1 a2 =
         a1 + a2
 
 "output.tags"
-with "--sort=no --extras=+r --fields=+rS"
+with "--options=NONE -o - --sort=no --extras=+r --fields=+rS input.elm"
 
 .. code-block:: tags
 

```